carbon-design-system/carbon
# fix(progress-bar): fix label accessibility violation

Fixes an accessiblity violation reported by the IBM Equal Access Accessibility Checker: A progress bar requires a label. The current implementation used a `<label>` element which is unsupported in combination with the target element (progress bar), because it's a `<div>` element. Therefore, the implemenation needs to change to use `aria-labelledby` instead.

#### Changelog

**Changed**

- Use `<span>` instead `<label>` for label
- Update tests

#### Testing / Reviewing

- Ensure the label is still shown with no visual regression
- Ensure there are no violations reported by the IBM Equal Access Accessiblity Checker
